src/main/java/org/junitpioneer/jupiter/DefaultTimeZone.java

```diff
@@ -22,7 +22,7 @@
  * returned by {@link java.util.TimeZone#getDefault()} for a test execution.
  *
  * <p>The {@link java.util.TimeZone} to set as the default {@code TimeZone} is
- * configured bei specifying the {@code TimeZone} ID as defined by
```

Comment:
Is this change related to this PR? (don't get me wrong, I appreciate the effort to clean things up as their found).

Reply:
If you wouldn't mind, and I promise I'll merge it immediately (are you signed up for HacktoberFest?).  As I noted above, I **do** appreciate people who help improve documentation!

Reply:
There you go: https://github.com/junit-pioneer/junit-pioneer/pull/147 ✌️ Will shortly rebase and update this branch.